index.js

```diff
@@ -1,17 +1,109 @@
 import fetch from 'node-fetch';
 // Recommend using node-fetch for those familiar with JS fetch
 
-const COLORS = 'https://nt-cdn.s3.amazonaws.com/colors.json';
+const COLORS_URL = 'https://nt-cdn.s3.amazonaws.com/colors.json';
 
 /**
+ * Retrieves colors JSON from the remote server
+ * @returns Promise
+ */
+const fetchData = async () => {
+  try {
+    const result = await fetch(COLORS_URL);
+    if (!result.ok) {
+      throw new Error("ERROR_FETCHING_COLOR_DATA:", result.status);
+    }
+
+    return result.json();
+  } catch(error) {
+    throw error;
+  }
+};
+
+
+/**
+ * Compares two name strings to determine if one includes the other
+ * 
+ * @param name string to compare against
+ * @param otherName search string for comparison 
+ * @returns boolean
+ */
+const compareNames = (name, otherName) => {
```

Comment:
some of your functions use destructuring in the signature and others don't. be consistent.